### PR TITLE
Add db.system attribute to the AWS SDK instrumentation.

### PR DESCRIPTION
**Pull request by rypdal**
## Changes

Added db.system attribute to the AWS SDK instrumentation and set value of the attribute equal to "dynamodb" for all DynamoDBv2 service related requests.

### CODE REVIEW

**Comment by Oberon00** [comment]
To add context here: db.system is a required attribute. It is even the only unconditionally required one. https://github.com/open-telemetry/opentelemetry-specification/blob/v1.12.0/specification/trace/semantic_conventions/database.md#connection-level-attributes

File: src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs [hunk 1]
```diff
@@ -143,6 +131,11 @@ private Activity ProcessBeginRequest(IExecutionContext executionContext)
                     activity.SetTag(AWSSemanticConventions.AttributeAWSRegion, region ?? AWSSDKUtils.DetermineRegion(client.ServiceURL));
                 }
 
+                if (AWSServiceHelper.IsDbService(service))
```
[the remark this answers is not in the document]
Comment (rypdal) [reply src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs]: Right. The method should have more specific name. Renamed to IsDynamoDbService. Also moved this code to the AddRequestSpecificInformation.

**Review by Oberon00 — APPROVED** [review]
LGTM, though later on I think the ParameterAttributeMap will have to be extended to support multiple attributes, and also arbitrary functions to get them instead of just property names (such a function could then also return a hard-coded constant like "dynamodb" here.

**Review by srprash — APPROVED** [review]
LGTM. Thanks!

**Comment by rypdal** [comment]
Thank you @Oberon00  and @srprash !

**Pull request merged**